datasource/keyvaultsecret/test-fixtures/template.pkr.hcl

```
@@ -0,0 +1,47 @@
+
+variable "keyvault_id" {
+    type=string
+}
+variable "client_id" {
+    type=string
```

I think these can default to the environment, is easier to have them set already than setting the PKR_VAR_client_id. What do you think? 
```suggestion
    type=string
    default = "${env("AZURE_CLIENT_ID")}"
```